build-config.yaml

```diff
@@ -1,6 +1,6 @@
 # Options intended to be common for all developers.
 
-wasm-size-limit: 15.97 MiB
+wasm-size-limit: 16.05 MiB
```

Comment:
I'm not sure. My initial guess would be that some code optimizations are no longer being applied because the searcher is no longer a fixed struct within the project, but a trait. Do we have a good way to debug Wasm size changes?

Reply:
I'm trying to have a look with `twiggy`.